<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialogClass</class>
 <widget class="QDialog" name="AddItemDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add MCQ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="questionGroupbox">
     <property name="title">
      <string>Question</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="questionRequiredCheckbox">
        <property name="text">
         <string>Required</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shuffleOptionsCheckbox">
        <property name="text">
         <string>Shuffle Options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addOptionBtn">
        <property name="text">
         <string>Add Option</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="1">
         <widget class="QComboBox" name="optionsCombobox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="optionsLabel">
          <property name="text">
           <string>Options</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gradingGroupbox">
     <property name="title">
      <string>Grading</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="pointValueLabel">
        <property name="text">
         <string>Point Value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="pointValueSpinbox">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="correctAnswerLabel">
        <property name="text">
         <string>Correct Answer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="correctAnswerCombobox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="imageGroupbox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="imageLinkLabel">
        <property name="toolTip">
         <string>The link from where the image will be downloaded.</string>
        </property>
        <property name="text">
         <string>Link</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="imageLinkLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="altTextLabel">
        <property name="toolTip">
         <string>The text that will appear in place of the image if the image couldn't be downloaded.</string>
        </property>
        <property name="text">
         <string>Alternative Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="altTextLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="imageAlignmentLabel">
        <property name="text">
         <string>Alignment</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="imageAlignmentCombobox">
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="imageWidthLabel">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="imageWidthSpinbox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="addItemDialogBtnBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
